src/pytest_plugins/test_filler/test_filler.py

```diff
@@ -315,7 +333,7 @@ def __init__(self, *args, **kwargs):
 
 @pytest.fixture(scope="function")
 def state_test(
-    request, t8n, fork, engine, reference_spec, eips, fixture_collector
+    request, t8n, fork, engine, reference_spec, eips, fixture_collector, base_test_config
```

Comment:
I think this should also be passed to the `blockchain_test` fixture? Currently, filled `blockchain_test` tests do not differ and still contain the `engineNewPayload` field.

Reply:
Yes, I missed this completely! I checked only one state test when I was verifying the output.